zio-http/shared/src/main/scala/zio/http/Server.scala

```diff
@@ -331,11 +336,18 @@ object Server extends ServerPlatformSpecific {
 
   def serve[R](
     httpApp: HttpApp[R],
+    logStartingServer: Boolean = true,
   )(implicit trace: Trace): URIO[R with Server, Nothing] =
     install(httpApp) *> ZIO.never
```

Comment:
Maybe it would be better to just add logging in this method, one log with something like "starting the server..." before call to `install(httpApp)` and other with "server started" after that call? I think it's not reasonable to add it to `install` itself. What do you think?